data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/model/ExportObjectKey.java

```diff
@@ -29,18 +33,21 @@ public class ExportObjectKey {
 
     public static ExportObjectKey fromString(final String objectKeyString) {
 
-        final String[] parts = objectKeyString.split("/");
-        if (parts.length != 6) {
+        final String[] parts = objectKeyString.split(S3_PATH_DELIMITER);
+        if (parts.length < 5) {
```

Comment:
any reason this change ?

Reply:
The previous code doesn't take into account where prefix has multiple levels (eg, "outer/inner") or is empty (eg, "").